src/main/java/Tasks.java

```diff
@@ -1,22 +1,31 @@
 public class Tasks {
     private int length = 0;
     String[] list = new String[100];
```

Comment:
Try to avoid magic numbers! Can consider assigning LIST_SIZE to 100 so it is clearer for the reviewer/reader